src/preamble_minimal.js

```diff
@@ -108,7 +108,7 @@ if (!ENVIRONMENT_IS_PTHREAD) {
 #if USE_PTHREADS
 } else {
 #if MODULARIZE
-  updateGlobalBufferAndViews({{{EXPORT_NAME}}}.buffer);
+  updateGlobalBufferAndViews(Module.buffer);
```

Comment:
Why is this changed?

Reply:
Without this it fails closure compiler with `MyModule` not defined.  I think this is because closure runs before modularization so at the point of closure compiler `MyModule` (the EXPORT_NAME) does not exist.

However `Module.buffer` works just fine here because the modularized code starts with

```
function(MyModule) {                                                             
  MyModule = MyModule || {};                                                     
                                                                                 
                                                                                 
                                                                                 
var Module = MyModule;  
```

Reply:
I see, sounds good. And closure will remove the extra var anyhow.